golang/cosmos/x/vstorage/vstorage_test.go

```diff
@@ -153,7 +153,7 @@ func doTestSet(t *testing.T, method string, expectNotify bool) {
 			// TODO: Fully validate input before making changes
 			// args:        []interface{}{[]string{"foo", "X"}, []interface{}{42, "new"}},
 			args:        []interface{}{[]interface{}{42, "new"}},
-			errContains: ptr("path"),
+			errContains: ptr("key"),
```

Comment:
Hmm, I think we want error messages from vstorage to continue referencing "path" rather than "key".

Reply:
Yeah I'm really not sure how to best do that if we want to keep combining the types.

Reply:
Fair enough; if there's no interception point then the benefits still outweigh the drawbacks.